game/templates/game/avatar.html

```
@@ -0,0 +1,191 @@
+{% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)